network/p2p/conduit/conduit.go

```diff
@@ -5,32 +5,18 @@ import (
 	"fmt"
 
 	"github.com/onflow/flow-go/model/flow"
-	"github.com/onflow/flow-go/module/component"
-	"github.com/onflow/flow-go/module/irrecoverable"
 	"github.com/onflow/flow-go/network"
 )
 
 // DefaultConduitFactory is a wrapper around the network Adapter.
 // It directly passes the incoming messages to the corresponding methods of the
 // network Adapter.
 type DefaultConduitFactory struct {
-	*component.ComponentManager
 	adapter network.Adapter
 }
 
 func NewDefaultConduitFactory() *DefaultConduitFactory {
-	d := &DefaultConduitFactory{}
-	// worker added so conduit factory doesn't immediately shut down when it's started
-	cm := component.NewComponentManagerBuilder().
-		AddWorker(func(ctx irrecoverable.SignalerContext, ready component.ReadyFunc) {
-			ready()
-
-			<-ctx.Done()
-		}).Build()
-
-	d.ComponentManager = cm
-
-	return d
```

Comment:
Conduit Factory no longer is going to run a `gRPC` server, we instead moved the server to `CorruptibleNetwork`, hence refactoring `ConduitFactory` from a `Component` to a `Module` (that no longer needs to be `ReadyDoneAware`).